dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/MethodAccessNode.java

```diff
@@ -90,4 +95,22 @@ public int hashCode() {
     public Collection<Node> getOperands() {
         return Collections.singletonList(receiver);
     }
+
+    /**
+     * Retrieves the modifiers of the method.
+     *
+     * @return the set of modifiers of the method
+     */
+    public Set<Modifier> getMethodModifiers() {
+        return method.getModifiers();
+    }
+
+    /**
+     * Check the method is a static or not.
```

Comment:
Same changes as for field access.
We usually use "check" if the method does some checks and raises errors. Here you get a boolean result.

Reply:
Done.